src/eureka/S4_generate_lightcurves/generate_LD.py

```diff
@@ -24,8 +28,31 @@ def exotic_ld(meta, spec):
     - July 2022, Eva-Maria Ahrer
         Initial version based on exotic_ld documentation.
     '''
+    
+    log.writelog("...using exotic-ld package...",
+                 mute=(not meta.verbose))
+    
+    # Check if exotic-ld directory includes csv files
+    exotic_ld_files = glob.glob(meta.exotic_ld_direc + '/**/*.flx',
```

Comment:
To try and make it possible to run Eureka on Windows, we've been using `os.sep` instead of `'/'` as Windows uses `'\'`